docs/docs/cmd/planner/task/task-details-get.md

```diff
@@ -0,0 +1,24 @@
+# planner task details get
+
+Retrieve the details of the specified planner task
+
+## Usage
+
+```sh
+m365 planner task get [options]
+```
+
+## Options
+
+`--id <id>`
```

Comment:
Looking at how the option is defined in the command, we're missing the short `-i` here